src/Microsoft.Health.Fhir.Core/Logging/Metrics/DefaultMetricEmitter.cs

```diff
@@ -0,0 +1,28 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using Microsoft.Health.Fhir.Core.Logging.Metrics;
+
+namespace Microsoft.Health.Fhir.Core.Logging.Metrics
+{
+    /// <summary>
+    /// <see cref="DefaultMetricEmitter"/> does not emit any metrics.
+    /// It's a default implementation to support the case where no metrics are required.
+    /// </summary>
+    public sealed class DefaultMetricEmitter : IFhirMetricEmitter
```

Comment:
Would it makes sense to have a real metric implementation that works with OSS? https://learn.microsoft.com/en-us/dotnet/core/diagnostics/metrics-instrumentation#get-a-meter-via-dependency-injection

Reply:
Added real implementations using IMeterFactory.